pastel/client.go

```diff
@@ -464,6 +464,27 @@ func (client *client) RegisterActionTicket(ctx context.Context, request Register
 	return txID.TxID, nil
 }
 
+func (client *client) ActivateActionTicket(ctx context.Context, request ActivateActionRequest) (string, error) {
+	var txID struct {
+		TxID string `json:"txid"`
+	}
+
+	params := []interface{}{}
+	params = append(params, "activate")
+	params = append(params, "action")
+	params = append(params, request.RegTxID)
+	params = append(params, fmt.Sprint(request.BlockNum))
+	params = append(params, fmt.Sprint(request.Fee))
+	params = append(params, request.PastelID)
+	params = append(params, request.Passphrase)
+
+	// command : tickets activate action "txid-of-action-reg-ticket" called_at_height-from_action-reg-ticket fee "PastelID-of-the-caller" "passphrase"
+	if err := client.callFor(ctx, &txID, "tickets", params...); err != nil {
+		return "", errors.Errorf("failed to call activate NFT ticket: %w", err)
```

Comment:
wrong message: should be `failed to call activate action ticket`

Reply:
fixed.